localization/yabloc/yabloc_image_processing/schema/graph_segment.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for graph_segment",
+  "type": "object",
+  "definitions": {
+    "graph_segment": {
+      "type": "object",
+      "properties": {
+        "target_height_ratio": {
+          "type": "number",
+          "description": "height on the image to retrieve the candidate road surface",
+          "default": 0.85
+        },
+        "target_candidate_box_width": {
+          "type": "number",
+          "description": "size of the square area to search for candidate road surfaces",
+          "default": 15
+        },
+        "pickup_additional_graph_segment": {
+          "type": "boolean",
+          "description": "if this is true, additional regions of similar color are retrieved",
+          "default": true
+        },
+        "similarity_score_threshold": {
+          "type": "number",
+          "description": "threshold for picking up additional areas",
+          "default": 0.8
+        },
+        "sigma": {
+          "type": "number",
+          "description": "parameter for cv::ximgproc::segmentation::GraphSegmentation",
+          "default": 0.5
+        },
+        "k": {
+          "type": "number",
+          "description": "parameter for cv::ximgproc::segmentation::GraphSegmentation",
+          "default": 300.0
+        },
+        "min_size": {
+          "type": "number",
+          "description": "parameter for cv::ximgproc::segmentation::GraphSegmentation",
+          "default": 100.0
+        }
+      },
+      "required": [
+        "target_height_ratio",
+        "target_candidate_box_width",
+        "pickup_additional_graph_segment",
+        "similarity_score_threshold",
+        "segmentation_sigma",
+        "segmentation_k",
+        "segmentation_min_size"
```

Comment:
There are some errors in my environment.
I am not sure why it can pass GitHub CI, but is it as expected?
```
~/autoware/src/universe/autoware.universe/localization/yabloc$ find -wholename '*/schema/*.schema.json' -printf '%p: ' -execdir bash -c 'check-jsonschema --schemafile "$1" ../config/"${1:2:-12}"*.param.yaml' bash '{}' + 
./yabloc_common/schema/ll2_decomposer.schema.json: ./yabloc_common/schema/ground_server.schema.json: ok -- validation done
./yabloc_image_processing/schema/graph_segment.schema.json: ./yabloc_image_processing/schema/segment_filter.schema.json: ./yabloc_image_processing/schema/undistort.schema.json: Schema validation errors were encountered.
  ../config/graph_segment.param.yaml::$./**.ros__parameters: 'segmentation_sigma' is a required property
  ../config/graph_segment.param.yaml::$./**.ros__parameters: 'segmentation_k' is a required property
  ../config/graph_segment.param.yaml::$./**.ros__parameters: 'segmentation_min_size' is a required property
./yabloc_pose_initializer/schema/camera_pose_initializer.schema.json: ok -- validation done
./yabloc_monitor/schema/yabloc_monitor.schema.json: ok -- validation done
./yabloc_particle_filter/schema/camera_particle_corrector.schema.json: ./yabloc_particle_filter/schema/gnss_particle_corrector.schema.json: ./yabloc_particle_filter/schema/predictor.schema.json: ok -- validation done
```

Reply:
Sorry  It is my fault :bow:

Reply:
Now I fixed it . Really sorry :pray: